envoy/Makefile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2017 Authors of Cilium
+# Copyright 2017-2019 Authors of Cilium
```

Comment:
Do we still need to have this file? :thinking:

Reply:
@jrajahalme ^

Reply:
I think we need it. See the `install` target at the end of this file. It takes care of unpacking `cilium-envoy` binary from the Docker image and installing it.